numpy/testing/_private/_pytester.py

```diff
@@ -0,0 +1,60 @@
+"""
+Generic test utilities.
+
+"""
+
+from __future__ import division, print_function, absolute_import
+
+import os
+import re
+import sys
+
+
+__all__ = ['PytestTester']
+
+
+class PytestTester(object):
+    """
+    Pytest test runner entry point.
+    """
+
+    def __init__(self, module_name):
+        self.module_name = module_name
+
+    def __call__(self, label="fast", verbose=1, extra_argv=None, doctests=False,
+                 coverage=False, tests=None):
+        import pytest
+
+        module = sys.modules[self.module_name]
+        module_path = os.path.abspath(module.__path__[0])
+
+        pytest_args = ['-l']
+
+        if doctests:
+            raise ValueError("Doctests not supported")
+
+        if extra_argv:
+            pytest_args += list(extra_argv)
+
+        if verbose and int(verbose) > 1:
```

Comment:
Any reason for the int coercion here?

Reply:
Don't know yet. I will be working on the file and just wanted to get it in start with.